Document the "Show agentless resources” toggle in the Fleet UI settings (#3760)
elastic/docs-content: reference/fleet/fleet-settings.md

## Advanced settings [fleet-advanced-settings]

On the **{{fleet}}** > **Settings** page, you can also configure {{fleet}} to automatically delete unenrolled agents or to display agentless resources for inspection and diagnostics purposes.

### Delete unenrolled agents [delete-unenrolled-agents-setting]

After an {{agent}} has been unenrolled in {{fleet}}, a number of documents about the agent are retained just in case the agent needs to be recovered at some point. You can choose to have all data related to an unenrolled agent deleted automatically.

Note that this option can also be enabled by adding the `xpack.fleet.enableDeleteUnenrolledAgents: true` setting to the [{{kib}} settings file](/get-started/the-stack.md).

To enable automatic deletion of unenrolled agents:

1. Go to **{{fleet}}** > **Settings**.
2. In the **Advanced Settings** section, enable the **Delete unenrolled agents** option.

### Show agentless resources [show-agentless-resources-setting]

```{applies_to}

> stack: ga 9.1.6

Author: This feature was added in 9.2 and it was backported to 9.1.6, so I assume having an `applies_to` for 9.1.6 also entails later versions as well.

Reviewer: I think this is correct

applying `stack` here does match our other documentation for agentless, but this is confusing to me because agentless is only available in ECH and serverless, it is not available for self-managed flavors (no matter if it's ECE, ECK, or regular stack)

but that's a larger cleanup effort so I'm fine with matching the rest of the documentation for now

serverless: ga
```

> If you have [agentless integrations](/solutions/security/get-started/agentless-integrations.md) deployed, you can enable the **Show agentless resources** option to display related resources such as agentless agents and policies for inspection and diagnostics purposes. This setting is stored locally, and it's only visible to you.

Author: I’d like to keep the second part to highlight the main purpose of the toggle.

```suggestion
If you have [agentless integrations](/solutions/security/get-started/agentless-integrations.md) deployed, you can enable the **Show agentless resources** option to display agentless agents and policies in {{fleet}} for inspection and diagnostics purposes. This setting is stored locally, and it's only visible to you.
```
